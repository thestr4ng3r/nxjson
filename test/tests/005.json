// array
[1,2,3,"last"]