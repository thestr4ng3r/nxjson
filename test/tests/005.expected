[
  1
  2
  3
  "last"
]
